### PR TITLE
v1.4.0

### PR DESCRIPTION
**Pull request by remnev**

### CODE REVIEW

**Comment by apsavin** [comment]
Why not 1.3.1? It's not a feature, it's a bugfix (extra redirect is a bug), isn't it?

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bem-stat-counters",
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (apsavin) [inline bower.json]: Remove this field, it's deprecated and ignored by bower
https://github.com/bower/spec/blob/master/json.md#version

**Comment by remnev** [comment]
It wasn't a bug. GA provided source code without strict https. Yandex metrica too.
But now they provide with.

**Comment by apsavin** [comment]
Well, if we tag it as 1.4.0 users with 1.3.* will not get this update. I think, it's wrong.

File: bower.json (hunk 1 above)
Comment (remnev) [inline bower.json]: It will be fixed through https://github.com/bem-contrib/stat-counters/issues/50

**Comment by remnev** [comment]
@zxqfox what's your opinion?

**Comment by zxqfox** [comment]
Well, I guess it's more like patch than minor, since it's not a addition to public API and it fixes broken functionality. ;-)

**Comment by remnev** [comment]
ok, updated

**Pull request merged**